components/commons/Navbar/Navbar.tsx

```diff
@@ -1,9 +1,24 @@
-import { FC } from "react";
-import { NavbarProps as Props } from "./Navbar.types";
-import styles from "./styles/Navbar.module.scss";
+import { FC } from 'react'
+import { NavbarProps as Props } from "./Navbar.types"
+import styles from "./styles/Navbar.module.scss"
 
 const Navbar: FC<Props> = ({}) => {
```

Comment:
```suggestion
const Navbar: FC = () => {
```
Change it like this as suggested